### DISCUSSION TITLE
Photo Animation Feature?

### DISCUSSION THREAD

**Discussion by Lockie85** [photoprism/photoprism#4250] [General]
Hey,

I was wondering if photoprism had a feature or similar feature to selecting a number of photos and animating them into a nice video? For example, you might take images of your babies first month and create a nice animation video of each day. Does such a thing exist?

Many Thanks

**Comment by lastzero** [comment]
It would certainly be cool if we had time to implement this :)

**Reply by Lockie85** [reply]
Should i put it in as a proper feature request, or is it not very likely to occur? Sorry, not sure if you're reply is saying nice idea but we're to busy or something else? I'm new here :D

**Reply by lastzero** [reply]
To be honest, looking at how many feature requests are already in line, we'd need a miracle (or a contribution).
